rct229/rules/section21/section21rule17.py

```diff
@@ -0,0 +1,99 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_baseline_system_types import get_baseline_system_types
+from rct229.utils.std_comparisons import std_equal
+from rct229.schema.config import ureg
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-1",
+    "SYS-5",
+    "SYS-7",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-1A",
+    "SYS-7A",
+    "SYS-11.2A",
+    "SYS-12A",
+]
+
+BOILER_RATED_CAPACITY_300000 = 300000 * ureg("Btu/hr")
```

Comment:
I would prefer that the names of the constants not be dependent on the choice of units.  Here the 300000 assumes Btu/hr is used.  How about `BOILER_RATED_CAPACITY_LOW_LIMIT` and `BOLER_RATED_CAPACITY_HIGH_LIMIT` or other terms that make more sense to you.

Reply:
Great point! Fixed.